src/libostree/ostree-core.h

```diff
@@ -271,10 +271,30 @@ typedef enum {
  * This is most useful in concert with `OSTREE_COMMIT_META_KEY_REF_BINDING`,
  * as it more strongly binds the commit to the repository and branch.
  *
+ * If you want to bind to multiple collections-ids, use OSTREE_COMMIT_META_KEY_COLLECTION_REFS_BINDING instead.
+ *
  * Since: 2018.6
  */
 #define OSTREE_COMMIT_META_KEY_COLLECTION_BINDING "ostree.collection-binding"
 
+/*
+ * OSTREE_COMMIT_META_KEY_COLLECTION_REFS_BINDING:
```

Comment:
Repeating this comment here (from elsewhere in my review) so it doesn’t get lost: it would be good to guarantee that the elements in this binding array are ordered alphabetically, to make searching/sorting/reproducibility of commits easier. Probably best to sort by collection ID and then by ref name? (i.e. The same as formatting all the tuples as `$collection_id:$ref_name` and sorting by that.)

Reply:
Guarantee how? This is already done by the changes in commit. Do you mean adding this to the docs?

Reply:
> Do you mean adding this to the docs?

Yup. So it’s an API guarantee which can be relied on by anyone reading this key, and has to be stuck to by anyone writing it.